src/main.rs

```diff
@@ -31,6 +32,8 @@ fn main() {
         (@arg timing: --time "Prints milliseconds after each stage")
         (@arg verify: -v --verify "Checks proof validity")
         (@arg verify_markup: -m --("verify-markup") "Checks comment markup")
+        (@arg verify_definitions: --("verify-definitions") [EXCLUSIONS]
```

Comment:
If I only set` a default value for `--verify-definitions`, then verifying definitions will occur every time!

Maybe this should be two parameters:
* `--verify-definitions` - do the verification
* `--definition-exclusions` - a list of label globs to exclude from definition checking when it occurs. Eventually this will disappear once the .mm dastabases have $j information so the information is in the databases themselves.

Reply:
That's right!
Yes, I think two options would do for now.

Reply:
> I suppose you will keep this PR as a draft and only merge once you have a working solution?

Yes, though my notion of "working solution" is (1) does *something* useful and (2) appears to be a useful starting point.

Reply:
Okay, will add (for now).